db/schema.rb

```diff
@@ -92,7 +92,7 @@
 
   create_table "assignments", force: :cascade do |t|
     t.string "name"
-    t.datetime "deadline", null: false
+    t.datetime "deadline", precision: nil, null: false
```

Comment:
Database changes are unrelated, please remove this.

Reply:
alright

Reply:
@salmoneatenbybear This isn't address yet